src/twincam.h

```diff
@@ -5,6 +5,7 @@
 
 struct options {
   unsigned long camera = 0;
+  bool camera_selected = false;
```

Comment:
I'd like to keep one variable per option, so if we change:

`unsigned long camera = 0;`

to

`long camera = -1;`

if (camera < 0) // we never explicitly set the camera

Makes sense?

Reply:
Yeah, that makes more sense. I like your solution more. We'll go with that. Thanks